docs/UserGuide.md

```diff
@@ -40,72 +39,134 @@ Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a task: `add`
+Adds a task to the FlexiTrack.<br>
+Format: `add [task title] < by/ [deadline] >`
 
-> Persons can have any number of tags (including 0)
+Examples: 
+* `add CS2103 tutorial 3 by/ Saturday`
+* `add CS2103 tutorial 3 by/ 22 Oct`
+
+#### Adding an event: `add`
+Adds a task to the FlexiTrack.<br>
+Format: `add [event title] from/ [starting time] to/ [ending time]`
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `add Bintan trip from/ Saturday to/ Sunday`
+* `add CS2103 Lecture from/ Friday 2pm to/ 4pm `
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
+#### Adding or changing deadline of a task : `deadline`
+Add or change the deadline of a task in FlexiTrack.<br>
+Format: `deadline [list number] [new deadline]`
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
+Examples: 
+* `deadline 3 10 October`
+* `deadline 12 Friday 5pm`
+
+#### Finding a task or an event containing any keyword in their title: `find`
+Finds a task ot an event whose title contain any of the given keywords.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+> * The search is case sensitive. e.g `soccer` will not match `Soccer`
+> * The order of the keywords does not matter. e.g. `soccer dinner` will match `dinner soccer`
+> * Only the task/event title is searched.
+> * Only full words will be matched e.g. `socc` will not match `soccer`
+> * Task or event matching at least one keyword will be returned (i.e. `OR` search).
+    e.g. `soccer` will match `soccer training`
```

Comment:
Consider yourself as a user, what kind of the search function is better to you? :)